.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Install Linux driver dependencies
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
-          sudo apt-get install libuv1 libuv1-dev
+          sudo DEBIAN_FRONTEND=noninteractive apt-get -y install libuv1 libuv1-dev default-jre
```

Comment:
what´s the reason for having to provide the  `DEBIAN_FRONTEND=noninteractive` and the `default-jre` ?

Reply:
@pfreixes default-jre need for java keytool command (used in tests for generate SSL certs for cassandra) DEBIAN_FRONTEND=noninteractive for install dependencies for default-jre which require interactive answers